packages/jest/src/builders/jest/jest.impl.ts

```diff
@@ -61,33 +25,47 @@ function run(
 ): Observable<BuilderOutput> {
   options.jestConfig = path.resolve(context.workspaceRoot, options.jestConfig);
 
-  const tsJestConfig = {
-    tsConfig: path.resolve(context.workspaceRoot, options.tsConfig)
-  };
-
-  // TODO: This is hacky, We should probably just configure it in the user's workspace
-  // If jest-preset-angular is installed, apply settings
-  try {
-    require.resolve('jest-preset-angular');
-    Object.assign(tsJestConfig, {
-      stringifyContentPathRegex: '\\.(html|svg)$',
-      astTransformers: [
-        'jest-preset-angular/build/InlineFilesTransformer',
-        'jest-preset-angular/build/StripStylesTransformer'
-      ]
-    });
-  } catch (e) {}
-
-  // merge the jestConfig globals with our 'ts-jest' override
-  const jestConfig: { globals: any } = require(options.jestConfig);
+  const jestConfig: {
+    transform: any;
+    globals: any;
+  } = require(options.jestConfig);
   const globals = jestConfig.globals || {};
-  Object.assign(globals, {
-    'ts-jest': {
-      ...(globals['ts-jest'] || {}),
-      ...tsJestConfig
+
+  // assume they don't use both 😬
```

Comment:
I don't think this should ever be the case, but just in case maybe we should log out a warning? I think anyone using `ts-jest` would use the babel option that they provide: https://kulshekhar.github.io/ts-jest/user/config/babelConfig

Reply:
Probably a good idea, I'll do that.